From 2f27df5fe92e33d9c4a9322ed2acfaf17f453167 Mon Sep 17 00:00:00 2001
From: Murali Karicheri <m-karicheri2@ti.com>
Date: Mon, 20 Mar 2017 14:17:58 -0400
Subject: [PATCH 3/3] ARM: dts: k2g-ice: dp83867: configure 4 level strap mode
 1

The K2G ICE EVM uses 4 level strap mode 1 to configure the dp83867
Phy. This patch add the DT parameter to configure the phy in
mode 1.

Signed-off-by: Murali Karicheri <m-karicheri2@ti.com>
---
 arch/arm/dts/k2g-ice.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/k2g-ice.dts b/arch/arm/dts/k2g-ice.dts
index 4224439..05790a1 100644
--- a/arch/arm/dts/k2g-ice.dts
+++ b/arch/arm/dts/k2g-ice.dts
@@ -85,6 +85,7 @@
 		ti,tx-internal-delay = <DP83867_RGMIIDCTL_250_PS>;
 		ti,fifo-depth = <DP83867_PHYCR_FIFO_DEPTH_8_B_NIB>;
 		ti,min-output-impedance;
+		ti,strap-mode = <DP83867_4LEVEL_STRAP_MODE1>;
 	};
 };
 
-- 
1.9.1

